src/Components/CreateImageWizard/steps/Oscap/OscapProfileInformation.tsx

```diff
@@ -126,4 +175,8 @@ export const OscapProfileInformation = (): JSX.Element => {
   );
 };
 
+OscapProfileInformation.defaultProps = {
+  allowChangingCompliancePolicy: false,
+};
+
```

Comment:
This can be removed as the default value of the prop is set directly on line 33. The warning in the test outputs then should go away.

Reply:
ohhh ok

Reply:
Hmm I don't see how it compiles without it ._.

Reply:
oh nvm found it, was missing a `?`